src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -62,22 +61,22 @@ export default () =>
       const receiver = control.contextReceiver("context-received");
       await control.openMockApp(openApp.c.name, undefined, "fdc3.instrument");
       await control.validateReceivedContext(await receiver, "fdc3.instrument");
-      await closeMockAppWindow(AOpensBWithContext2Test);
+      await control.closeMockApp(AOpensBWithContext2Test);
     });
 
     const AOpensBWithContext3Test = "(AOpensBWithContext3) Can open app B from app A with context and AppMetadata (name and appId) as target, app B adds generic listener";
     it(AOpensBWithContext3Test, async () => {
       const receiver = control.contextReceiver("context-received");
       await control.openMockApp(openApp.c.name, openApp.c.id, "fdc3.instrument");
       await control.validateReceivedContext(await receiver, "fdc3.instrument");
-      await closeMockAppWindow(AOpensBWithContext3Test);
+      await control.closeMockApp(AOpensBWithContext3Test);
     });
 
     const AOpensBMalformedContext = `(AOpensBMalformedContext) App B listeners receive nothing when passing a malformed context`;
     it(AOpensBMalformedContext, async () => {
       const receiver = control.contextReceiver("context-received", true);
       await control.openMockApp(openApp.f.name, undefined, undefined, true, true);
       await receiver;
-      await closeMockAppWindow(AOpensBMalformedContext);
```

Comment:
should be closeMockAppWindow?

Reply:
I remember now why I'm calling the closeMockApp function via the interface implementation instead of directly from the utils folder. It's because the common tests require the function to be called from the 1.2 utils file or the 2.0 utils file depending on whether 1.2 or 2.0 tests are being run.

using the interface, the tests need to be able to freely switch between the two versions of the function

Reply:
see comment above